src/components/feature/auth/login/Login.jsx

```diff
@@ -25,10 +25,8 @@ const Login = () => {
 
   return (
     <section className="section login">
-      <Link to="/">
-        <button className="arrow-back-icon" type="button">
-          <ArrowBack />
-        </button>
+      <Link to="/" className="arrow-back-icon">
```

Comment:
zrób tak....<Link {...{to: CONSTANTS.ROUTE_PATHS.HOME, className: "arrow-back-icon"}}>
        <ArrowBack />
      </Link> czytelnie, co? :)